app/auth/email.py

```diff
@@ -1,5 +1,6 @@
 from flask import current_app, render_template, url_for, redirect, flash
 from flask_mail import Message
+from flask_login import current_user
```

Comment:
```suggestion
```
No. I removed this for a reason - abstraction makes it much easier to maintain. Please use `current_user` in `views.py` only.